#! /usr/bin/env python
# GftDetector dynamic reconfigure

PACKAGE='ccny_rgbd'

from dynamic_reconfigure.parameter_generator_catkin import *


gen = ParameterGenerator()
                                                                    
gen.add("n_features", int_t, 0, "Number of feautures requested", 400, 1, 1000) 
gen.add("min_distance", int_t, 0, "Minimum distance between features (pixels)", 1, 0, 15) 

exit(gen.generate(PACKAGE, "dynamic_reconfigure_node", "GftDetector"))

